applications/CoSimulationApplication/python_scripts/data_transfer_operators/copy.py

```diff
@@ -19,13 +21,13 @@ def TransferData(self, from_solver_data, to_solver_data, transfer_options):
 
         transfer_options_list = transfer_options.GetStringArray()
 
+        # the order is IMPORTANT here!
```

Comment:
Yes .. is this order shown to user at some point ?

Reply:
this is meant in terms of implementation, for the user it does not matter

I will add an explanatory comment in the next PR